pavelib/assets.py

```diff
@@ -165,7 +165,7 @@ def collect_assets(systems, settings):
     `settings` is the Django settings module to use.
     """
     for sys in systems:
-        sh(django_cmd(sys, settings, "collectstatic --noinput > /dev/null"))
+        sh(django_cmd(sys, settings, "collectstatic --clear --noinput > /dev/null"))
```

Comment:
I believe that's what `REQUIRE_DEBUG` setting in `lms/envs/devstack.py` does.  Maybe we should disable `PIPELINE_JS_COMPRESSOR` in those settings as well?